src/modules/Dashboard/features/Participant/Assignments/ByParticipant/ByParticipant.tsx

```diff
@@ -0,0 +1,11 @@
+import { useTranslation } from 'react-i18next';
+
+import { AssignmentsTab } from '../AssignmentsTab';
+
+const ByParticipant = () => {
+  const { t } = useTranslation();
+
+  return <AssignmentsTab />;
+};
+
+export default ByParticipant;
```

Comment:
Personally I think this is fine, but it seems to introduce an ESLint warning

```suggestion
import { AssignmentsTab } from '../AssignmentsTab';

const ByParticipant = () => <AssignmentsTab />;

export default ByParticipant;

```